docs/user/tutorials/01-40-10-gcp-redis-instance.md

```diff
@@ -0,0 +1,157 @@
+# Create Redis Instances in Google Cloud
+
+Learn how to create a Redis instance in Google Cloud and connect to it.
+
+## Prerequisites  <!-- {docsify-ignore} -->
+
+- You have the Cloud Manager module added.
+- Install [Redis CLI](https://redis.io/docs/latest/develop/tools/cli/).
+
```

Comment:
```suggestion
```
There is no need to install redis-cli (or anything for that matter).
The redis-cli is already available on the pod that is being run in the example.